src/System.Security.Cryptography.Pkcs/tests/CertLoader.cs

```diff
@@ -83,6 +86,8 @@ public X509Certificate2 TryGetCertificateWithPrivateKey()
         public abstract byte[] PfxData { get; }
         public abstract string Password { get; }
 
+        public static object SyncObject { get; } = new object();
```

Comment:
Based on the other conversation in the other PR, I don't think we need this anymore.